CHANGELOG.md

```diff
@@ -7,6 +7,14 @@
 * Provide applications with the ability to override default logic for "restore defaults".
 This allows complex and device-specific sub-apps to perform more targeted and complete clearing of
 user state.  See new overridable method `HoistAppModel.restoreDefaultsAsync` for more information.
+* New `ColumnAutosizeOptions` config bundles existing column-specific autosize properties along with
+additional `bufferPxOverride` property, which overrides `GridAutosizeOptions.bufferPx`
```

Comment:
Did we consider losing the 'override' suffix? It seems clear enough to me that setting a property on the Column's autosizeOptions would override the property on the GridModel, and would keep the name consistent.

Reply:
I did consider losing the override suffix, but in the 2 methods above, I found the code confusing to read the same variable `bufferPx` off of either options (from grid) or autosizeOptions (from column). I like your refactor above, but it seems easier to read with the `override`. Also, I thought it might be ambiguous whether the column's bufferPx were additive vs replacing.